### ISSUE TITLE
why are events seperated with a dash

### ISSUE THREAD

**Issue by depuits** [ankurk91/vue-trumbowyg#36]
**I'm submitting a ...**  (check one with "x")
```
[ ] Bug report => search github for a similar issue or PR before submitting
[ ] Feature request
[x] Other, documentation issue
```

**Tell about your platform**
* Trumbowyg version : 2.x.x
* Vue.js version : 2.6.10
* This package version : 3.4.2

**Current behavior**

To subscribe to events you have to use the plain js event but separate the prefix and event with a dash.

**Expected behavior**

I'm not sure if this is intended and/or has a technical reason, but it would make more sense if the events had the same name. So if you go look in the documentation it would be the same. 

Otherwise I would suggest adding a note in the readme to differentiate this difference more clearly. Now the readme only refers to the plain js events and later on shows an example from which you can deduce that you need to add the dash.

**Comment by ankurk91** [comment]
I followed vue js documentation to name the events.
https://vuejs.org/v2/guide/components-custom-events.html#Event-Names

I will a note about native vs vue events

**Comment by depuits** [comment]
That's a good reason but because the event name are already in lowercase it wasn't really necessary to use the kebab-case. Because of compatibility is would be best to just update the documentation to make it clear for new users. Keep up the great work :)

**Comment by ankurk91** [comment]
Should be closed with v3.5.0
